_topic_maps/_topic_map.yml

```diff
@@ -3314,7 +3314,7 @@ Topics:
     File: virt-live-migration-limits
   - Name: Migrating a virtual machine instance to another node
     File: virt-migrate-vmi
-  - Name: Migrating a virtual machine over a dedicated secondary network
+  - Name: Migrating a virtual machine over a dedicated additional network
     File: virt-migrating-vm-on-secondary-network
```

Comment:
Should we align the file name with the title?

Reply:
No. This is an assembly. If we change the file name, we will have to set up redirects. So we try to avoid changing the names of assembly files unless it is absolutely necessary. In this case, it is better to keep the file name and just change the title.

Reply:
No change.